lib/src/model/lobby/game_seek.dart

```diff
@@ -113,4 +117,70 @@ sealed class GameSeek with _$GameSeek {
     if (variant != null) 'variant': variant!.name,
     if (ratingRange != null) 'ratingRange': '${ratingRange!.$1}-${ratingRange!.$2}',
   };
+
+  factory GameSeek.fromJson(Map<String, dynamic> json) => _$GameSeekFromJson(json);
+}
+
+class RecentGameSeekPrefs implements Serializable {
```

Comment:
It would be better to use an `IList` and a `@freezed` class here (we do that in the rest of the app).